src/sidebar/store/create-store.js

```diff
@@ -52,6 +52,7 @@ export default function createStore(modules, initArgs = [], middleware = []) {
       allReducers[module.namespace] = createReducer(module.update);
       allSelectors[module.namespace] = {
         selectors: module.selectors,
+        rootSelectors: module.rootSelectors || {},
```

Comment:
Given the defaulting to empty object here, leads me to wonder what happens if a store module doesn't have any selector...

Reply:
Currently this will cause an error - We'll either need to add an empty `selectors` object to modules or change this code to default to `{}` if the need arises.